packages/client/src/components/Modals/AddAgency.vue

```diff
@@ -107,8 +105,8 @@ export default {
       formData: {
         name: null,
         abbreviation: null,
-        warningThreshold: null,
-        dangerThreshold: null,
+        warningThreshold: 2,
```

Comment:
Set to 14

Reply:
Yes, please set the default warning threshold to 14 days, and default danger threshold to 7 days.